src/components/SocialLinks-Color/index.js

```diff
@@ -20,23 +22,23 @@ const SocialLinksColor = () => {
             target="_blank"
             rel="noreferrer"
           >
-            <img height="30px" src={forum_icon} alt="forum" />
+            <StaticImage height={30} src={forum_icon} alt="forum" objectFit="contain"/>
           </a>
           <a
             className="mail_icon"
             href="mailto:community@layer5.io"
             target="_blank"
             rel="noreferrer"
           >
-            <img height="30px" src={mail_icon} alt="mail" />
+            <StaticImage height={30} src={mail_icon} alt="mail" objectFit="contain"/>
           </a>
           <a href="https://slack.layer5.io/" target="_blank" rel="noreferrer">
-            <img className="slack" height="30px" src={slack_icon} alt="slack" />
+            <StaticImage className="slack" height={30} src={slack_icon} alt="slack" objectFit="contain"/>
```

Comment:
Since, you are already defining `object-fit` property in the stylesheet I think it can be omitted here